### PR TITLE
Feat/bank feeds api

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

Significant changes to Bank Feeds docs

## Type of change

Please delete options that are not relevant.

- [X] New document(s)/updating existing
- [X] Fixes
- [X] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: docs/bank-feeds/setup.md [hunk 1]
```diff
@@ -1,177 +1,98 @@
...
-<Tabs>
+Once you decide to build this flow with Bank Feeds API, you need to configure Codat accordingly. Let's go through these requirements in detail.
 
-<TabItem value="Request URL" label="Request URL">
+### Enable the product
 
-You can create a company by using the [Create company](/bank-feeds-api#/operations/create-company) endpoint:
+1. Open the <a href="https://app.codat.io" target="_blank">Codat Portal</a> and sign in.
+2. Click on **Settings > Organizational settings > Products**.
+3. In the list of products, find _Bank Feeds API_ and click **Enable**. Then, follow the on-screen prompt.
```
Comment (dcoplowe) [inline docs/bank-feeds/setup.md:56]: We should follow Lending API documentation? 
- Level 2 title and before configure product
- Should be titled 'Enable Bank Feeds API'

File: docs/bank-feeds/setup.md [hunk 2]
```diff
@@ -1,177 +1,98 @@
...
-    "name": "{CompanyName}"
-}
-```
+In the <a href="https://app.codat.io" target="_blank">Codat Portal</a>, navigate to **Settings > Integrations** and click **Manage integrations**. Next, click **Manage** next to the specific integration you want to enable and set it up to serve as a data source for the product. 
 
-</TabItem >
+<IntegrationsList integrations={bankfeedsIntegrations}/>
 
-<TabItem value="Response" label="Response">
+Some of these integrations may require enhanced setup specific to bank feeds. We walk you through these in our integration-specific instructions in the _Manage integrations_ section of our Bank Feeds API documentation.
```
Comment (dcoplowe) [inline docs/bank-feeds/setup.md:64]:
```suggestion
Some of these integrations require additional setup specific to bank feeds. We walk you through these in our integration-specific instructions in the _Manage integrations_ section of our Bank Feeds API documentation. 
```

File: docs/bank-feeds/setup.md [hunk 3]
```diff
@@ -1,177 +1,98 @@
...
-    "created": "2023-09-06T09:13:35.8188152Z"
-}
-```
+### Authorization flow
 
-</TabItem >
+As part of using Bank Feeds API, you will need your customers to authorize your access to their data. To do so, use Link - our pre-built, conversion-optimized, and white-labelled authorization flow. 
 
-</Tabs>
+We recommend you fully embed the Link auth flow in your experience by using our [Embedded Link](/auth-flow/authorize-embedded-link) SDK in your front-end code. You can also choose our out-of-the-box [Hosted Link](/auth-flow/authorize-hosted-link) auth flow option to get up and running as quick as possible.
```
Comment (dcoplowe) [inline docs/bank-feeds/setup.md:70]: We should be opinionated about what type of link they use.

File: docs/bank-feeds/setup.md [hunk 4]
```diff
@@ -1,177 +1,98 @@
...
-    "platform": "",
-    "redirect": "https://link.codat.io/company/77921ff9-2491-4dfe-b23b-ff28f3e31e4f",
-    "dataConnections": [],
-    "created": "2023-09-06T09:13:35.8188152Z"
-}
-```
+### Authorization flow
 
-</TabItem >
+As part of using Bank Feeds API, you will need your customers to authorize your access to their data. To do so, use Link - our pre-built, conversion-optimized, and white-labelled authorization flow.
```
Comment (dcoplowe) [inline docs/bank-feeds/setup.md:68]:
```suggestion
As part of using Bank Feeds API, you will need your customers to authorize your access to their data. To do so, use [Embedded Link](https://codat-docs-git-feat-bank-feeds-api-codat.vercel.app/auth-flow/authorize-embedded-link) - our pre-built, embeddable, conversion-optimized, and white-labelled authorization flow. 
```

File: docs/bank-feeds/setup.md (hunk 3 above)
Comment (dcoplowe) [inline docs/bank-feeds/setup.md:70]:
```suggestion
```

File: docs/bank-feeds/setup.md [hunk 5]
```diff
@@ -1,177 +1,98 @@
...
 
+Codat supports a range of [webhooks](/using-the-api/webhooks/core-rules-types) to help you manage your data pipelines. In the <a href="https://app.codat.io" target="_blank">Codat Portal</a>, navigate to **Settings > Webhooks > Rules** and click **Create new rule** to set up the following webhook and get the most out of Bank Feeds API:
 
-| Accounting platform | platformKey |
-| ---  | ---  |
-| Quickbooks Online Bankfeeds | `hcws` |
-| Xero | `gbol` |
-| FreeAgent | `fbrh` |
-| Sage Bank Feeds | `olpr` |
+- [Push operation status has changed](/using-the-api/webhooks/core-rules-types#push-operation-status-has-changed)
```
Comment (dcoplowe) [inline docs/bank-feeds/setup.md:82]: I've recently seen a bank feeds webhook event in our data. Is this event type missing? Bank Feed account status change

File: docs/bank-feeds/create-account.md [hunk 6]
```diff
@@ -1,140 +1,269 @@
...
-:::
 
-Codat categorizes bank accounts into either credit or debit types for standardization. 
+```
+
+:::tip Authorize your API calls
+
+Remember to [authenticate](/using-the-api/authentication) when making calls to our API. Navigate to **Developers > API keys** in the Portal to pick up your authorization header.
+
+:::
```
Comment (dcoplowe) [inline docs/bank-feeds/create-account.md:59]: Love this.

File: docs/bank-feeds/create-account.md [hunk 7]
```diff
@@ -1,140 +1,269 @@
...
 ---
 
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem"
 
-To establish a bank feed with your customer's accounting software, you'll first need to [set up source accounts](/bank-feeds-api#/operations/create-source-account). These accounts serve as representations of the company's financial accounts within the Codat platform. 
+## Overview
 
-During the subsequent mapping process, these source accounts can be linked to corresponding accounts in the customer's accounting software.
+When implementing your bank feed solution, you need to create several key objects required by Codat's Bank Feeds API: a [company](../terms/company), its connection to a data source, and a source account.
```
Comment (dcoplowe) [inline docs/bank-feeds/create-account.md:13]:
```suggestion
When implementing your bank feed solution, you need to create your customer as a [company](../terms/company) in Codat before registering their accounting platform as a [connection](../terms/connection). Something about source accounts.
```

File: docs/bank-feeds/create-account.md [hunk 8]
```diff
@@ -1,140 +1,269 @@
...
-To establish a bank feed with your customer's accounting software, you'll first need to [set up source accounts](/bank-feeds-api#/operations/create-source-account). These accounts serve as representations of the company's financial accounts within the Codat platform. 
+## Overview
 
-During the subsequent mapping process, these source accounts can be linked to corresponding accounts in the customer's accounting software.
+When implementing your bank feed solution, you need to create several key objects required by Codat's Bank Feeds API: a [company](../terms/company), its connection to a data source, and a source account. 
 
+You can see how these elements fit together and where they sit in the overall bank feeds process on the diagram below.
 
-:::note UK specific requirements
+```mermaid
```
[the remark this answers is not in the document]
Comment (pzaichkina) [reply docs/bank-feeds/create-account.md:17]: will ask Jake on Monday 👌

File: docs/bank-feeds/integrations/qbo/mapping.md [hunk 9]
```diff
@@ -1,9 +1,8 @@
 ---
-title: "QuickBooks Online bank feeds mapping"
+title: "Map accounts in QuickBooks Online"
 description: "Push bank transaction data into your customers' accounting platforms with an automated feed"
-sidebar_label: QBO mapping
```
[the remark this answers is not in the document]
Comment (pzaichkina) [reply docs/bank-feeds/integrations/qbo/mapping.md:4]: there will be more pages... so no, i ll leave it as is

**Review by dcoplowe — APPROVED** [review]

**Pull request merged**